docs/source/processing.rst

```diff
@@ -134,6 +134,111 @@ The :func:`datasets.Dataset.shard` takes as arguments the total number of shards
 
 This method can be used to slice a very large dataset in a predefined number of chunks.
 
+
+Renaming, removing, casting and flattening columns
+--------------------------------------------------
+
+Renaming a column: ``rename_column``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This method renames a column in the dataset, and move the features associated to the original column under the new column name.
```

Comment:
```suggestion
This method renames a column in the dataset, and move the features associated to the original column under the new column name. This operation will fail if the new column name already exists.
```

This is probably obvious and not worth mentioning. Eager to hear your views @lhoestq